src/gsv.rs

```diff
@@ -0,0 +1,66 @@
+use crate::common;
+use crate::satellite::Satellite;
+
+use crate::Source;
+/// Satellites in views including the number of SVs in view, the PRN numbers, elevations, azimuths, and SNR values.
+#[derive(Debug, PartialEq, Clone)]
+pub struct GSV {
+    /// The total number of GSV messages for the current data.
+    pub total_messages_number: u8,
+    /// The message number (1 to the total number of messages) for the current GSV sentence.
+    pub message_number: u8,
+    /// Total number of satellites in view.
+    pub sat_in_view: u8,
+    /// Information about first SV.
+    pub sat_info_1: Option<Satellite>,
```

Comment:
Satellites may be more comfort to use as an array. Looks better to make it as private static array with size up to 4, for example [Satellite; MAX_SATELITES_PER_MESSAGE] and additional field of actual array size.

It opens a way to write getter method returning satellites as a immutable slice borrowing GSV structure.

Reply:
Done.